tests/test_rio_features.py

```diff
@@ -21,8 +21,8 @@ def test_mask(runner, tmpdir, basic_feature, basic_image_2x2,
     output = str(tmpdir.join('test.tif'))
 
     result = runner.invoke(
-        features.mask,
-        [pixelated_image_file, output, '--geojson-mask', '-'],
+        main_group,
+        ['mask', pixelated_image_file, output, '--geojson-mask', '-'],
```

Comment:
I changed all the `invoke` calls like this so that the commands would have a `ctx.obj` with `aws_session` in it.